### ISSUE TITLE
Identificar posibles patrocinadores para enviar propuesta de patrocinio

### ISSUE THREAD

**Issue by ecabrerar** [JavaDominicano/conference#137]
Se desea crear un listado de posibles empresas que pueden ser patrocinadores.
De cada empresa se desea saber Nombre de la empresa, Nombre del Contacto Principal, Correo del contacto principal y teléfono del contacto principal. 

El listado será trabajado aquí https://docs.google.com/spreadsheets/d/1pS0AWl9B0pxT4sCJBihkxsH_s-SSJbiggqKCfyHJD3E/edit?usp=sharing

**Comment by xTryHard** [comment]
✋🏾

**Comment by ecabrerar** [comment]
@xTryHard alguna novedad con este tema?

**Comment by xTryHard** [comment]
Voy a actualizar hoy. Me faltan algunos correos y teléfonos (no he actualizado la lista oficial, la del documento)

**Comment by xTryHard** [comment]
Iteración sobre el issue. 
Sería interesante llegar a 15 o 20 propuestos. Tengo otros anotados pero me gustaría conseguir más detalles de contacto 

Qué opina? @ecabrerar

**Comment by ecabrerar** [comment]
> Iteración sobre el issue. Sería interesante llegar a 15 o 20 propuestos. Tengo otros anotados pero me gustaría conseguir más detalles de contacto
> 
> Qué opina? @ecabrerar

Estoy de acuerdo, voy a ver si consigo el documento de los contactos del barcamp 2022 para agregar algunos

**Comment by ecabrerar** [comment]
@xTryHard agregué unos cuantos, revisa a ver este documento a ver si se puede sacar algo más https://docs.google.com/spreadsheets/d/1CHhSS9Rfht2_07dpT0fwqBZyNYNp15CLGY-VMOlgEmk/edit?usp=sharing
